docs/modules/ROOT/pages/index.adoc

```diff
@@ -323,13 +325,36 @@ The `signer_payload` data structure MUST be presented to be signed by the _<<_cr
 
 IMPORTANT: If a data hash assertion is being used, the _<<C2PA claim generator>>_ MUST also follow the process described in xref:_interaction_with_data_hash_assertion[xrefstyle=full].
 
+==== Calculating the `expected_claim` value
+
+If the *<<_identity_assertion,identity assertion>> generator* chooses to include the `expected_claim` value in `signer_payload`, it MUST be computed as follows:
+
+. Create a valid _<<C2PA claim>>_ data structure that includes completed references to all _<<_c2pa_assertion,C2PA assertions>>,_ *including* all *<<_identity_assertion,identity assertion>>.*
+. For the specific *<<_identity_assertion,identity assertion>>* being computed, replace the `hash` value with 32 zero-value (`0x00`) bytes.
+. Serialize the _<<C2PA claim>>_ data structure using link:++https://www.rfc-editor.org/rfc/rfc8949.html#name-core-deterministic-encoding++[Section 4.2.1, “Core deterministic encoding,”] of RFC 8949: Concise binary object representation.
+. Compute a SHA-256 hash of the serialized _<<C2PA claim>>_ data structure.
+
+The resulting hash should be used as the `expected_claim` value.
+
+IMPORTANT: Since this algorithm replaces the `hash` value for exactly one *<<_identity_assertion,identity assertion>>,* this implies that only one *<<_identity_assertion,identity assertion>>* per _<<C2PA Manifest>>_ may provide an `expected_claim` value and that this *<<_identity_assertion,identity assertion>>* MAY NOT be referenced by any other *<<_identity_assertion,identity assertion>>.*
+
+==== Calculating the `expected_claim_generator` value
+
+If the *<<_identity_assertion,identity assertion>> generator* chooses to include the `expected_claim_generator` value in `signer_payload`, it MUST be computed as follows:
+
+. Identify signing certificate chain that will be used for the expected _<<C2PA claim generator>>._ This chain should include certificates for all intermediate certificate authorities.
```

Comment:
All intermediates, but not the root?

Reply:
This follows precedent set by the C2PA itself, which also omits the root.

Reply:
Having the cert chain minus the root allows using a trust list to complete the chain validation, right. Isn't this however a duplicated chain? The claim generator signature will contain this chain, but the purpose of this cert information in the id assertion is just to make sure it's matching with the claim's. For this purpose, an end-entity thumbprint would be sufficient and reduce duplication.